### ISSUE TITLE
Camera repair - AI "station alerts" window pops up for crew

### ISSUE THREAD

**Issue by 00aquir** [yogstation13/Yogstation#7642]
- "Current station alerts" window pops up  every single time when re-activating cameras as human (and maybe when AI or borgs are also checking the alerts?)
- Happened in several rounds on different days.

**Comment by MegaEmpirical** [comment]
Just had this, fuck this

**Comment by 00aquir** [comment]
even to this day..

**Comment by 00aquir** [comment]
still happens sometimes

**Comment by 00aquir** [comment]
yes, often it keeps popping up and of course it makes the camera re-activation even harder for every camera because it brings the focus away from your hand

**Comment by Bibby0110** [comment]
Happens for me in the armory

**Comment by 00aquir** [comment]
still happening aaa
